highlevel/util/perf_metrics.py

```diff
@@ -30,7 +30,7 @@ async def print_performance_metrics() -> None:
         active_tasks = len(tasks)
 
         vmem = psutil.virtual_memory()
-        LOGGER.get().info(
+        LOGGER.get().debug(
```

Comment:
why? maybe increase the time interval if it bothers you?  
Also does not seem related to this diff

(